### ISSUE TITLE
Problem with embed with fullpage.js

### ISSUE THREAD

**Issue by carstenheisterkamp** [distantnative/embed#86]
Hi, I use[ fullpage.js](https://github.com/alvarotrigo/fullPage.js) for my portfolio page, which works nicely with kirby and I want to use it with the [embed plugin](https://github.com/distantnative/embed), which also works, but with the following hickup

* The video starts independently from what I set for autostart in the config or code.
* While plays the cover is still displayed and the video is playing under the cover image
* When I click the cover the whole page loads again inside the oembed / vimeo iframe.
Not sure if this is related to my dev server (working on a localhost) or fullpagejs or my lack of php skills... ;-)
Basically I am using the code from the embed github tutorial.

Any help appreciated.
Thanks
C

**Comment by carstenheisterkamp** [comment]
Can be closed. Problem was a misconfiguration of my fullpage.js
